worlds/sc2wol/docs/setup_en.md

```diff
@@ -16,6 +16,7 @@ to obtain a config file for StarCraft 2.
    - Linux users should also follow the instructions found at the bottom of this page 
      (["Running in Linux"](#running-in-linux)).
 2. Run ArchipelagoStarcraft2Client.exe.
+   - macOS users should instead follow the instructions found at (["Running in macOS"](#running-in-macos)) for this step only.
```

Comment:
Oh, I see. The style matches too good here though. The `()` for Linux is because it says to look at the bottom, not because it's a link. Other links don't have `()` around them.

```suggestion
   - macOS users should instead follow the instructions found at ["Running in macOS"](#running-in-macos) for this step only.
```